src/test/common/steps/send-code-steps.js

```diff
@@ -23,13 +23,33 @@ Then(/^Text and Email options are displayed on the send code page$/, async funct
 
   expect(text).to.include(TEXT_LABEL)
   expect(text).to.include(EMAIL_LABEL)
+
+  const textHint = await pages.sendCode.getTextHint()
+  const textHintText = await textHint.getText()
+  expect(textHintText).to.include(PHONE_NUMBER)
+
+  const emailHint = await pages.sendCode.getEmailHint()
+  const emailHintText = await emailHint.getText()
+  expect(emailHintText).to.include(EMAIL_ADDRESS)
 })
 
 Then(/^I am informed that I need to select an option for send code$/, async function () {
   await assertErrorHeaderTextPresent(pages.sendCode)
   await assertSendCodePresent()
 })
 
+Then(/^The change-text link points to the phone number page$/, async function () {
+  const changeLink = await pages.sendCode.getChangeTextLink()
+  const linkHref = await changeLink.getAttribute('href')
+  expect(linkHref).to.be.equal(`${pages.url}/phone-number`)
```

Comment:
Can we get the link from a page object please? Same with below too.

Reply:
done